scripts/globals/npc_util.lua

```diff
@@ -7,25 +7,34 @@ npcUtil = {};
 -- Picks a new position for an NPC and excluding the current position.
 -- INPUT: npc = npcID, position = 2D table with coords: index, {x, y, z}
 -- RETURN: table index
-function npcUtil.pickNewPosition(npc, positionTable)
-
+function npcUtil.pickNewPosition(npc, positionTable, allowCurrentPosition)
     local npc = GetNPCByID(npc)
     local positionIndex = 1 -- Default to position one in the table if it can't be found.
     local tableSize = 0
+    local allowCurrentPosition = allowCurrentPosition or false;
```

Comment:
I read this is the lua way to pass optional parameters, so other scripts using it don't need to be modified